graphql_compiler/compiler/filters.py

```diff
@@ -452,6 +452,43 @@ def _process_in_collection_filter_directive(filter_operation_info, location, con
     return blocks.Filter(filter_predicate)
 
 
+@scalar_leaf_only(u'not_in_collection')
+@takes_parameters(1)
+def _process_not_in_collection_filter_directive(filter_operation_info, location, context, parameters):
```

Comment:
You could break the line right at the comma and put `parameters` on the next line, aligned under `filter_operation_info`.